src/main/java/com/jagrosh/vortex/commands/moderation/ModlogsCmd.java

```diff
@@ -21,6 +21,64 @@ public ModlogsCmd(Vortex vortex) {
         this.guildOnly = true;
     }
 
+    @Override
+    protected void execute(CommandEvent event) {
+        if (event.getArgs().equalsIgnoreCase("help")) {
+            event.replySuccess("This command is used to see a user's modlogs for the current server. Please mention a user or use a user ID to check their modlogs.");
+            return;
+        }
+
+        long id = CommandTools.getPossibleUserId(event.getArgs().trim());
+
+        if (id == -1) {
+            if (event.getArgs().trim().isEmpty()) {
+                id = event.getAuthor().getIdLong();
+            } else {
+                event.reply("Please mention someone or enter a valid user ID");
+                return;
+            }
+        }
+
+        List<Modlog> modlogs = Database.getAllModlogs(event.getGuild().getIdLong(), id);
+
+        int size = modlogs.size();
+
+        if (size == 0) {
+            event.reply("Could not find any modlogs for that user");
+            return;
+        }
+
+        EmbedBuilder[] embeds = new EmbedBuilder[size / 25 + 1];
+        for (int i = 0; i < embeds.length; i++) {
+            embeds[i] = new EmbedBuilder();
+        }
+
+        if (event.getJDA().getUserById(id) != null) {
+            embeds[0].setAuthor(String.format("%d modlog%s found for %s#%s (%d)", size, size == 1 ? "" : "s", event.getJDA().getUserById(id).getName(), event.getJDA().getUserById(id).getDiscriminator(), id), null, String.format(event.getJDA().getUserById(id).getAvatarUrl()));
```

Comment:
Note that getAvatarUrl() returns the URL only if the user has a custom pfp, otherwise it will return null if they still have a default pfp, while getEffectiveUrl() would return the default pfp url as well. This is a pattern in other methods in this library such as getName()/getEffectiveName() so just something to be aware of
```suggestion
            embeds[0].setAuthor(String.format("%d modlog%s found for %s#%s (%d)", size, size == 1 ? "" : "s", event.getJDA().getUserById(id).getName(), event.getJDA().getUserById(id).getDiscriminator(), id), null, String.format(event.getJDA().getUserById(id).getEffectiveAvatarUrl()));
```